projects/scullyio/ng-lib/src/lib/transfer-state/transfer-state.service.ts

```diff
@@ -60,7 +61,7 @@ export class TransferStateService {
         first()
       )
     ),
-    map((ev: NavigationEnd) => ev.url),
+    map((ev: NavigationEnd) => ev.urlAfterRedirects),
```

Comment:
I would prefer:
```ts
ap((ev: NavigationEnd) => ev.urlAfterRedirects || ev.url)
```
As Angular doesn't give an gurantee that the afterRedirects is allways filled.

Reply:
Alright, I'll change it